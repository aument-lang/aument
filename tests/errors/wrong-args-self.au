class A{}

def (self: A) fn() {
    fn(1,2,3);
}