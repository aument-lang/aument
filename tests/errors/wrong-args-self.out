parser error(4) in -: wrong number of arguments (expected 1, got 3)
4 |     fn(1,2,3);
        ^^
